local-cli/eject/eject.js

```diff
@@ -64,6 +66,15 @@ function eject() {
     process.exit(1);
   }
 
+  const appIcon = appConfig.icon;
+  if(!appIcon) {
```

Comment:
I decided to get rid of this whole if statement that makes the icon mandatory. If there is no icon it'll fall back to what it does already without this configuration